src/main/java/appeng/tile/inventory/AppEngCellInventory.java

```diff
@@ -3,15 +3,15 @@
 
 
 import net.minecraft.item.ItemStack;
+import net.minecraftforge.items.IItemHandlerModifiable;
 
 import appeng.api.storage.ICellInventory;
 import appeng.api.storage.ICellInventoryHandler;
 import appeng.util.inv.IAEAppEngInventory;
-import appeng.util.inv.IInternalItemHandler;
 import appeng.util.inv.filter.IAEItemFilter;
 
 
-public class AppEngCellInventory implements IInternalItemHandler
+public class AppEngCellInventory implements IItemHandlerModifiable
 {
 	private final AppEngInternalInventory inv;
 	private final ICellInventoryHandler handlerForSlot[];
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the array designator from the variable to the type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1197)